core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -197,14 +207,7 @@ class ZkUtils(val zkClient: ZkClient,
                               BrokerSequenceIdPath,
                               IsrChangeNotificationPath)
 
-  val securePersistentZkPaths = Seq(BrokerIdsPath,
```

Comment:
1) okay
2) where would I put such a comment ? You're right this is necessary to prevent this issue from happening in the future
3) agreed. Just wanted to keep things easy without refactoring code in other places